lib/GameConstants.cpp

```diff
@@ -86,7 +86,7 @@ const Artifact * ArtifactID::toArtifact(const ArtifactService * service) const
 
 si32 ArtifactID::decode(const std::string & identifier)
 {
-	auto rawId = VLC->modh->identifiers.getIdentifier("core", "artifact", identifier);
+	auto rawId = VLC->modh->identifiers.getIdentifier("map", "artifact", identifier);

```

Comment:
Looks like this needs a constant. SpecialScopes::MAP or something

Reply:
Also core scope might be put there as well

Reply:
Makes sense. Should I do that as part of this PR?

Reply:
I mean I do not expect it destabilize beta so we can do it now,

Reply:
Okay. In that case I'll add
- scope for core objects
- scope for in-game requests (e.g. requests fixed in this PR)
- scope for requests from map loader (for now - aliased to in-game scope, later - it should take use of map dependencies)